apps/webapp/src/components/swap/asset-out-selector.tsx

```diff
@@ -22,30 +22,25 @@ export const AssetOutSelector = ({ balances, setAssetOut, assetOut }: AssetOutSe
           </p>
         </div>
       </DialogTrigger>
-      <DialogContent className='max-w-[312px] bg-charcoal-secondary md:max-w-[400px]'>
```

Comment:
The max widths here are now the default, and the background is already set in the `DialogContent` component definition